app/views/admin/users/_member_index.html.erb

```diff
@@ -30,7 +30,7 @@
                 <div>
                   <%= render "admin/users/index/status_indicator", user: user %>
                 </div>
-                <span class="c-indicator ml-2 crayons-hover-tooltip" data-tooltip="<%= cascading_high_level_roles(user) %>">
+                <span class="c-indicator ml-2 crayons-hover-tooltip" data-tooltip="<%= format_role_tooltip(user) %>">
```

Comment:
```suggestion
                <span class="c-indicator ml-2 crayons-hover-tooltip crayons-hover-tooltip--right" data-tooltip="<%= format_role_tooltip(user) %>">
```

I only added this to the organizations before, but this instance appears on the right-aligned side too

Reply:
So, I'm in quite the situation right now: I'm actually unable to make this change locally, as my local branch ran into merge conflicts/didn't have the latest code and somewhere down the line while I was fixing everything, my branch essentially broke beyond repair and now I'm unable to push code to GH. 😬 That said, would you mind if I do it in a follow-up PR, @aitchiss?

Reply:
oh my days, what a way to start the week - poor you! And of course, no worries about a follow up 🙂